ft-sdk/src/auth/mod.rs

```diff
@@ -28,6 +28,15 @@ impl ProviderData {
             .get(key)
             .and_then(|v| serde_json::from_value(v.clone()).ok())
     }
+
+    /// get the first verified or unverified email address
```

Comment:
Mention in docs that it panics if no email is found. Better make it not panic, and return an `Option<String>`.

Reply:
done in https://github.com/FifthTry/ft-sdk/pull/52/commits/0a5ecdbf461ba61b8f2bbe73f5281385613a5bfb